src/data.py

```diff
@@ -46,7 +50,7 @@ def parse_eatery(data_json):
     phone = eatery.get('contactPhone', 'N/A')
     phone = phone if phone else 'N/A'  # handle None values
```

Comment:
`phone or 'N/A'` since `phone` shouldn't be falsy anyway

Reply:
not your code but just something i noticed

Reply:
phone can be `None` if in the dictionary `contactPhone: None`

Reply:
yeah i meant falsy but not `None`, the `or` will handle `None`s properly so it's just slightly cleaner